code/code/misc/riding.cc

```diff
@@ -1,4 +1,4 @@
-#include "handler.h"
+nclude "handler.h"
```

Comment:
Looks like a couple characters got deleted here, probably going to cause an error